src/dnd/DragArea.js

```diff
@@ -0,0 +1,88 @@
+import React, { Component } from 'react';
+import { DndContext } from './DndContext';
+
+class DragArea extends Component {
+  state = {
+    dragStart: false,
+    draggable: null,
+    x: 0,
+    y: 0,
+    lx: 0,
```

Comment:
rename to `lastX` or something else, probably

Let's not use not clear variables / fields